packages/imperative.ts

```diff
@@ -30,6 +30,7 @@ const config: IImperativeConfig = {
     rootCommandDescription: Constants.DESCRIPTION,
     defaultHome: Constants.HOME_DIR,
     envVariablePrefix: Constants.ENV_PREFIX,
+    webHelpLogoImgPath: require("find-up").sync("web-help/logo.png", {cwd: process.mainModule.filename}),
```

Comment:
I think in order for the logo to be included in the package, it needs to be listed under `files` in `package.json`? 

Also, every require/dependency added to initialization of the CLI decreases response time. Is `find-up` absolutely necessary here?

Reply:
Oops, I did forget to list the file in `package.json`. Thanks for noticing that, will fix next week.

I believe the location of the image file could vary based on whether the CLI repo has been cloned from GitHub or installed as a package from NPM. To eliminate the extra dependency, I suppose it would be possible to detect the environment the CLI is running in to determine where to look for the file, like Imperative does [here](https://github.com/zowe/imperative/pull/262/files#diff-f8e90a58045f1997cb2d7de60bcdf18bR124).

Reply:
Sounds good thanks.

Reply:
@tucker01 I removed usage of the `find-up` package and replaced what it was doing with a separate function (`tryFindWebHelpLogoImgPath`). Adding this function above the definition of `IImperativeConfig` feels wrong to me, but not sure where else to put it since I don't see a module for utility functions. I'm wondering if there's a better way to guide Imperative where to look for the logo image?